tensorboard/webapp/metrics/views/main_view/main_view_test.ts

```diff
@@ -395,6 +400,10 @@ describe('metrics main view', () => {
           sample: 0,
         },
       ]);
+      store.overrideSelector(
```

Comment:
Override getSortedRenderableCardIdsWithMetadata instead of getNonEmptyCardIdsWithMetadata and getCurrentRouteRunSelection. (same for almost all the other changes in this file, too)